### PR TITLE
Use object typehint when generating entities

### PR DESCRIPTION
**Pull request by HypeMC**
Use object typehint when adding object entity fields.

### CODE REVIEW

File: src/Util/ClassSourceManipulator.php [hunk 1]
```diff
@@ -1113,6 +1113,8 @@ private function getEntityTypeHint($doctrineType)
                 return '\\'.\DateInterval::class;
 
             case 'object':
+                return \PHP_VERSION_ID < 70200 ? null : 'object';
```
Comment (jrushlow) [inline src/Util/ClassSourceManipulator.php:1116]: This check should be done in the `PhpCompatUtil` -> https://github.com/symfony/maker-bundle/blob/main/src/Util/PhpCompatUtil.php

Something along the lines of `PhpCompatUtil::canUseObjectReturnType()`

**Review by jrushlow — CHANGES_REQUESTED** [review]
I like the idea! Anytime we can set explicit types, the better. We do however need to move the PHP version conditional logic into the `PhpCompatUtil::class`

File: src/Util/ClassSourceManipulator.php (hunk 1 above)
Comment (HypeMC) [reply src/Util/ClassSourceManipulator.php]: @jrushlow Done

Comment (HypeMC) [reply src/Util/ClassSourceManipulator.php]: @jrushlow Since the maker bundle now requires PHP 7.2.5+ I've removed all conditional logic.

**Review by jrushlow — APPROVED** [review]
Awesome! Thank you @HypeMC

**Pull request merged**